### ISSUE TITLE
livereload only reloads changes to the html file

### ISSUE THREAD

**Issue by worldspawn** [schickling/gulp-webserver#109]
Hi,

I might be misunderstanding what livereload does, but using the below setup I see changes to the html file be reflected, however changes to assets the html is loading are not reloaded. For example if I change a css file, or js file nothing happens. My simple test was to change the background color defined in body (fails) and change some text in the html documents (success). Should I be seeing changes to assets be automatically injected?

```
gulp.task('serve', ['watch'], function() {
    return gulp.src('./app')
      .pipe(plugins.webserver({
          livereload: {port: 8032, enabled: true},
          open: true,
          middleware: [serveStatic(__dirname + '/../')]
      }));
});
```

**Comment by alvincrespo** [comment]
Yeah I'm also a bit confused on this, I thought that livereload would check for .js files throughout the project?
